src/test/javascript/spec/component/course/course-overview.component.spec.ts

```diff
@@ -0,0 +1,257 @@
+                { provide: Router, useClass: MockRouter },
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(CourseOverviewComponent);
+                component = fixture.componentInstance;
+                courseService = TestBed.inject(CourseManagementService);
+                courseScoreCalculationService = TestBed.inject(CourseScoreCalculationService);
+                teamService = TestBed.inject(TeamService);
+                jhiWebsocketService = TestBed.inject(JhiWebsocketService);
+                fixture.detectChanges();
+            });
+    });
+
+    afterEach(() => {
+        component.ngOnDestroy();
+        sinon.restore();
+    });
+
+    it('Should call all methods on init', fakeAsync(() => {
+        const getCourseStub = stub(courseScoreCalculationService, 'getCourse');
+        const subscribeToTeamAssignmentUpdatesStub = stub(component, 'subscribeToTeamAssignmentUpdates');
+        const subscribeForQuizChangesStub = stub(component, 'subscribeForQuizChanges');
+        const adjustCourseDescriptionStub = stub(component, 'adjustCourseDescription');
+        const findOneForDashboardStub = stub(courseService, 'findOneForDashboard');
+        findOneForDashboardStub.returns(of(new HttpResponse({ body: course1, headers: new HttpHeaders() })));
+        getCourseStub.returns(course1);
+
+        component.ngOnInit();
+        tick(1000);
+
+        expect(getCourseStub).to.have.been.called;
+        expect(adjustCourseDescriptionStub).to.have.been.called;
+        expect(subscribeForQuizChangesStub).to.have.been.called;
+        expect(subscribeToTeamAssignmentUpdatesStub).to.have.been.called;
+    }));
+
+    it('Should call load Course methods on init', fakeAsync(() => {
+        const getCourseStub = stub(courseScoreCalculationService, 'getCourse');
+        const subscribeToTeamAssignmentUpdatesStub = stub(component, 'subscribeToTeamAssignmentUpdates');
+        const subscribeForQuizChangesStub = stub(component, 'subscribeForQuizChanges');
+        const adjustCourseDescriptionStub = stub(component, 'adjustCourseDescription');
+        const findOneForDashboardStub = stub(courseService, 'findOneForDashboard');
+        findOneForDashboardStub.returns(of(new HttpResponse({ body: course1, headers: new HttpHeaders() })));
+
+        component.ngOnInit();
+        tick(600);
+
+        expect(getCourseStub).to.have.been.called;
+        expect(adjustCourseDescriptionStub).to.have.been.called;
+        expect(subscribeForQuizChangesStub).to.have.been.called;
+        expect(subscribeToTeamAssignmentUpdatesStub).to.have.been.called;
+    }));
+
+    it('should set Long Description', () => {
+        component.longTextShown = false;
+
+        component.showLongDescription();
+
+        expect(component.courseDescription).to.equal('');
+        expect(component.longTextShown).to.equal(true);
```

Comment:
We usually use `.to.be.true` (Same with a few other `.equals(false/true)` below.)